recipes/bioconductor-cancermutationanalysis/build.sh

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
@ArneKr can you please remove the leading newlines here?
This should be fixed in the newest skeleton generator.